src/components/form/HdGoogleAutocomplete.vue

```diff
@@ -112,16 +104,8 @@ export default {
     t() {
       return merge(getMessages(this.lang), this.texts);
     },
-    isEmpty() {
-      return this.value === null || this.value === undefined || this.value === '';
-    },
-    fieldClasses() {
-      return {
-        'field--active': this.isActive,
-        'field--filled': !this.isEmpty,
-        'field--invalid': this.isValid === false,
-        'field--disabled': this.disabled,
-      };
+    isFilled() {
+      return this.value !== null && this.value !== undefined && this.value !== '';
```

Comment:
We are using `lodash` in the project, maybe we could use `_.isEmpty` to handle this

Reply:
`_.isEmpty` is for collections in general, so it wouldn't do the job for us here, for example `_.isEmpty(99)` would be `true`, which is not what we would expect.
And improvement would be to use `_.isNil` instead of checking for `null` and `undefined` separately, but we will still need the check for the empty string.
I remember having this discussion with the rest of the team a long time ago, and we said that keeping it as explicit as it is right now is easier to understand.
But we can also create our own helper to check if the form fields are filled.

Reply:
Oh, sorry about that, I didn't know about that conversation =/
And I'm used with `rambda` instead of `lodash` and `isEmpty(99)` in Rambda is `false` 😅 

But it is completely fine to keep explicit =D